client/src/components/questionsAnswers/QuestionCard.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import axios from 'axios';
+import PropTypes from 'prop-types';
+import AnswerCard from './AnswerCard.jsx';
+
+const { useState, useEffect } = React;
+
+function QuestionCard(props) {
+  const { question } = props;
+  const [answers, setAnswers] = useState([]);
+
+  useEffect(() => {
+    // const queryParams = {
```

Comment:
What is the purpose of the commented out lines (13-18)? Please consider omitting or if you leave it in, please clarify within the comment the purpose of 13-18.